src/IECoreHoudini/FromHoudiniGeometryConverter.cpp

```diff
@@ -876,6 +872,12 @@ DataPtr FromHoudiniGeometryConverter::extractStringData( const GU_Detail *geo, c
 	return data;
 }
 
+const IECore::InternedString &FromHoudiniGeometryConverter::groupPrimVarPrefix()
+{
+	static IECore::InternedString grp("__grp:");
```

Comment:
what is this prefix for? it seems really arbitrary... should it have `ie` in there somewhere?

Reply:
I wanted  a prefix so it was clear the primvar was from a houdini group attribute. How about renaming to *ieGroup* ?

Reply:
sounds good